map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -239,6 +237,33 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
                 return@setOnMarkerClickListener true
             }
             groundOverlayGenerated.onNext(true)
+
+            /**
+             * Add Lion icons.
+             * TODO:: fix bounds of map.
+             */
+            map.addMarker(MarkerOptions()
+                    .position(LatLng(41.879678006591391, -87.624091248446064))
+                    .icon(bitmapDescriptorFromVector(requireContext(), R.drawable.map_lion_1)))
+
+            map.addMarker(MarkerOptions()
+                    .position(LatLng(41.879491568164525, -87.624089977901931))
+                    .icon(bitmapDescriptorFromVector(requireContext(), R.drawable.map_lion_2)))
+
+        }
+    }
+
+    /**
+     * Builds BitmapDescriptor from vector drawable.
+     */
+    private fun bitmapDescriptorFromVector(context: Context, vectorResId: Int): BitmapDescriptor? {
```

Comment:
You should add an @DrawableRes annotation to this Int. Also might want to put this in the MarkerGenerators.kt file @agrosner proposed in https://github.com/art-institute-of-chicago/aic-mobile-android/pull/69